README.md

```diff
@@ -365,28 +366,34 @@ You can see the diff between the current DB schema and the ghsync schema here:
 Migrations reside in `database/migrations` and they need to be packed with go-bindata before being usable.
 To repack migrations you can use:
 
-```
+```shell
 make migration
 ```
 
 ### Testing
 
 To test, run:
 
-```
-GITHUB_TOKEN=<xxx> go test ./...
-```
+```shell
+# set your github personal access token (scopes 'read:org', 'repo')
+export GITHUB_TOKEN=<xxx>
```

Comment:
IMO we should use `GITHUB_TOKENS` as well here, wdyt? Or maybe use `GITHUB_TOKEN_TEST` or `TEST_GITHUB_TOKEN`.

Reply:
But [current test requires `GITHUB_TOKEN`](https://github.com/src-d/metadata-retrieval/blob/master/github/downloader_test.go#L84).

Reply:
Yup, I know I was just wondering whether we would like to uniform things. Now I need to set two environment variables `GITHUB_TOKEN` and `GITHUB_TOKENS`. But not a blocker, just an idea. \cc @kyrcha.

I'm approving this.

Reply:
I'll merge as it is, and we can discuss it in a new issue. We could also consider if the tests should run as many times as tokens are provided, or discard all but the first one.

Reply:
For the shake of less configuration GITHUB_TOKENS as a sole option seems fine for me. If you add only one it will be like before. I will update the tests when this new issue comes up.